packages/helpers/test/useMountedRef.spec.js

```diff
@@ -0,0 +1,29 @@
+import { renderHook } from "@testing-library/react-hooks";
+
+import useMountedRef from "../src/hooks/useMountedRef";
+
+describe("useMountedRef", () => {
+  test("should increment counter", () => {
+    const {
+      result: { current: mountedRef },
+      rerender,
+      unmount,
+    } = renderHook(() => {
```

Comment:
ohhh, so this is like you were coding your component and using the hook inside of it?

Reply:
Sort of